auslib/db.py

```diff
@@ -1465,7 +1465,13 @@ def _versionMatchesRule(self, ruleVersion, queryVersion):
         self.log.debug('ruleVersion: %s, queryVersion: %s', ruleVersion, queryVersion)
         if ruleVersion is None:
             return True
-        return version_compare(queryVersion, ruleVersion)
+        rulesVersionList = ruleVersion.split(",")
+        versionMatchesRule = False
+        for rule in rulesVersionList:
+            versionMatchesRule = version_compare(queryVersion, rule)
+            if versionMatchesRule:
+                break
+        return versionMatchesRule
```

Comment:
This is an improvement, but it can still be simplified by getting rid of the break and just returning from within the loop.